js/samples/advanced-csharp/Pages/Index.cshtml

```diff
@@ -112,7 +112,22 @@
             const token = await getImmersiveReaderTokenAsync(Canary);
             const subdomain = await getImmersiveReaderSubdomainAsync();
 
-            ImmersiveReader.launchAsync(token, subdomain, data);
+            // Learn more about options https://docs.microsoft.com/azure/cognitive-services/immersive-reader/reference#options
+            const options = {
+                cognitiveAppId: 'AdvancedSampleWebApp',
+                onExit: exitCallback
+            };
+
+            try {
```

Comment:
try/catch? Doesn't this need Promise handling?

Reply:
changed to promise handling, for the rest of pages too